src/quo2/screens/list_items/dropdown.cljs

```diff
@@ -0,0 +1,71 @@
+(ns quo2.screens.list-items.dropdown
+  (:require [quo.components.text :as text]
+            [quo.previews.preview :as preview]
+            [quo.react-native :as rn]
+            [quo2.components.list-items.dropdown :as quo2]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]))
+
+(def descriptor [{:label   "Icon"
+                  :key     :icon
+                  :type    :select
+                  :options [{:key   :main-icons/placeholder
+                             :value "Placeholder"}
+                            {:key   :main-icons/wallet
+                             :value "Wallet"}]}
+                 {:label "Disabled"
+                  :key   :disabled?
+                  :type  :boolean}
+                 {:label "Default item"
+                  :key   :default-item
+                  :type  :text}
+                 {:label "Border color"
```

Comment:
can you make it possible to disable the border here too? so that the designs can be verified.
Consider using a `select` option with none as an optiona nd some example colors

Reply:
Done